exonum-light-client/src/test/java/com/exonum/client/ExonumHttpClientBlocksIntegrationTest.java

```diff
@@ -82,6 +81,8 @@
 
 class ExonumHttpClientBlocksIntegrationTest {
 
+  private final String expectedBlocksPath = "api/explorer/v1/blocks";
```

Comment:
Isn't it equal to `BLOCKS`? Or did you inline it everywhere on purpose?

Reply:
Yes, now it's equal. But what if an error in the BLOCKS path? So I inlined it to the variable for removing duplicates